cea/demand/demand_writers.py

```diff
@@ -78,22 +81,72 @@ def write_to_csv(self, building_name, columns, hourly_data, locator):
 
 class HourlyDemandWriter(DemandWriter):
     """Write out the hourly demand results"""
+
     def __init__(self, gv):
         super(HourlyDemandWriter, self).__init__(gv)
 
     def write_to_csv(self, building_name, columns, hourly_data, locator):
         hourly_data.to_csv(locator.get_demand_results_file(building_name), columns=columns, float_format=FLOAT_FORMAT)
 
+    def write_totals_csv(self, building_properties, locator):
+        """read in the temporary results files and append them to the Totals.csv file."""
+        counter = 0
+        for name in building_properties.list_building_names():
+            temporary_file = locator.get_temporary_file('%(name)sT.csv' % locals())
+            if counter == 0:
+                df = pd.read_csv(temporary_file)
+                counter += 1
+            else:
+                df2 = pd.read_csv(temporary_file)
+                df = df.append(df2, ignore_index=True)
+        df.to_csv(locator.get_total_demand(), index=False, float_format='%.3f')
+
+        """read saved data of hourly values and return as totals"""
+        hourly_data_buildings = [pd.read_csv(locator.get_demand_results_file(building_name)) for building_name in
+                                 building_properties.list_building_names()]
+        return df, hourly_data_buildings
+
+        return df
+
 
 class MonthlyDemandWriter(DemandWriter):
     def __init__(self, gv):
         super(MonthlyDemandWriter, self).__init__(gv)
         self.vars_to_print = [['QEf', 'QHf', 'QCf', 'Ef'], [], []]
 
     def write_to_csv(self, building_name, columns, hourly_data, locator):
-        monthly_data = hourly_data[[x + '_kWh' for x in self.vars_to_print[0]]].groupby(
+        # get monthly totals and rename to MWhyr
+        monthly_data = hourly_data[[x + '_kWh' for x in self.vars_to_print[LOAD_VARS]]].groupby(
             by=[hourly_data.index.month]).sum() / 1000
+
         monthly_data = monthly_data.rename(
             columns=dict((x + '_kWh', x + '_MWhyr') for x in self.vars_to_print[LOAD_VARS]))
-        monthly_data['Name'] = building_name
-        monthly_data.to_csv(locator.get_demand_results_file(building_name), index=False, float_format=FLOAT_FORMAT)
+
+        peaks = hourly_data[[x + '_kWh' for x in self.vars_to_print[LOAD_VARS]]].groupby(
+                   by=[hourly_data.index.month]).max()
+        peaks = peaks.rename(
+            columns=dict((x + '_kWh', x + '0_kW') for x in self.vars_to_print[LOAD_VARS]))
+
+        monthly_data_new = monthly_data.merge(peaks, left_index=True, right_index=True)
+
+        monthly_data_new['Name'] = building_name
+        monthly_data_new.to_csv(locator.get_demand_results_file(building_name), index=False, float_format=FLOAT_FORMAT)
+
+
+    def write_totals_csv(self, building_properties, locator):
```

Comment:
Yes, we can add it. But i do not understand how the superclass work, is there another simpler structure for this?